src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -119,281 +119,363 @@
   </resheader>
   <data name="ColorScheme_Background.Text" xml:space="preserve">
     <value>Background</value>
+    <comment>The background color for text displayed on the screen. Users can select a color to use in this context.</comment>

```

Comment:
Sorry, these need to be more about the _string_, not about what the setting does. "This is the header for a control that lets the user select the background color for text displayed on the screen."